python/semantic_kernel/connectors/ai/anthropic/services/anthropic_chat_completion.py

```diff
@@ -154,6 +154,7 @@ async def _inner_get_streaming_chat_message_contents(
         self,
         chat_history: "ChatHistory",
         settings: "PromptExecutionSettings",
+        function_invoke_attempt: int = 0,
```

Comment:
When does a user want to pass in a value that's not 0?

Reply:
This `_inner_get_streaming_chat_message_contents` method is called for a normal chat completion and auto function calling -- so I'm providing a default value here so the caller in the non-auto function calling scenario doesn't have to pass anything in.

Reply:
But why would someone want to pass anything in?

Reply:
We synced offline about this - this is the inner API taking the new keyword arg.